### PR TITLE
96 remove modal

### PR DESCRIPTION
**Pull request by fadeomar**
#### Issues this PR relates to
#96
=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. 

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/components/Layout/Step/style.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  background: ${({ theme }) => theme.gradients.secondary};
+  width: 50%;
+  max-width: 260px;
+  align-self: normal;
+`;
+
+export const ContentHalf = styled.div`
+  flex: 1;
+  background: ${({ theme }) => theme.colors.white};
+  padding: ${({ theme: { spacings: s } }) =>
```
Comment (RamyAlshurafa) [inline client/src/components/Layout/Step/style.js:25]: wrong values, don't match figma

File: client/src/pages/Step/index.js [hunk 2]
```diff
@@ -34,14 +33,16 @@ function Step(props) {
     query: `(max-width: ${breakpoints.tablet})`,
   });
   return (
-    <Modal {...props}>
+    <>
```
Comment (RamyAlshurafa) [inline client/src/pages/Step/index.js:36]:
```suggestion
```

**Review by RamyAlshurafa — CHANGES_REQUESTED** [review]
- [ ] avoid fix numbers in px
image

File: client/src/components/Layout/Step/style.js (hunk 1 above)
Comment (fadeomar) [reply client/src/components/Layout/Step/style.js:25]: done

File: client/src/pages/Step/index.js (hunk 2 above)
Comment (fadeomar) [reply client/src/pages/Step/index.js:36]: done

**Review by thejoefriel — APPROVED** [review]

**Pull request merged**